speaker/main.go

```diff
@@ -72,7 +71,7 @@ func main() {
 		mlSecret        = flag.String("ml-secret-key", os.Getenv("METALLB_ML_SECRET_KEY"), "Secret key for MemberList (fast dead node detection)")
 		myNode          = flag.String("node-name", os.Getenv("METALLB_NODE_NAME"), "name of this Kubernetes node (spec.nodeName)")
 		port            = flag.Int("port", 7472, "HTTP listening port")
-		logLevel        = flag.String("log-level", "info", fmt.Sprintf("log level. must be one of: [%s]", strings.Join(logging.Levels, ", ")))
+		logLevel        = flag.String("log-level", "info", fmt.Sprintf("log level. must be one of: [%s]", logging.Levels.String()))
```

Comment:
Not sure I like this more or less than it was before. I'd probably add a custom function to the logging package as opposed to convert the slice to a string. WDYT?

Reply:
TBH, I'm not very happy with that code: in order to get the same string as before, I added a String() method to the []Level type, adding a custom type as well (`logging.levelSlice`). It's probably over engineering, but since Levels is not a []string, it cannot be passed to strings.Join(...).

We can go for a fmt.Sprintf(...) of levels Slice, getting a little different output, if it's not a big deal.

Reply:
Fair enough, let's keep it as it is.